### DISCUSSION TITLE
如何获取群成员的头衔？（群主设置的专属头衔）

### DISCUSSION THREAD

**Discussion by hxtscjk17373** [simple-robot/Discussions#53] [Q&A] [answered]
如题。。我能看到我要获取的头衔在这里，但是没有找到获取它的方法。
求解答，谢谢！
![图片](https://user-images.githubusercontent.com/77102146/210208673-f046b6ac-e188-4c1f-aa5f-c9969b1d9bf1.png)

**Comment by ForteScarlet** [comment, marked as answer]
see [component-mirai#61](https://github.com/simple-robot/simbot-component-mirai/pull/61)
监听Mirai组件事件或判断群成员对象 `Member` 为 `MiraiMember` 类型时可使用相关操作。

```java
Member member = group.getMember(...);
if (member instanceof MiraiMember miraiMember) {
     String title = miraiMember.getSpecialTitle();
     miraiMember.setSpecialTitle(...);
}
```

**Reply by hxtscjk17373** [reply]
明白了~非常感谢！
